mne/io/tests/test_pick.py

```diff
@@ -94,9 +94,12 @@ def test_pick_refs():
         if len(pick) > 0:
             pick_info(info, pick)
 
+    msg = 'Removing %d compensators' % len(info['comps'])
     for pick in (picks_meg, picks_mag):
         if len(pick) > 0:
-            pytest.raises(RuntimeError, pick_info, info, pick)
+            with catch_logging() as log:
+                pick_info(info, pick, verbose=True)
+            assert msg in log.getvalue()
```

Comment:
If `assert 'Removing {} compensators'.format(len(info['comps']) in log.getvalue()` does not fit in a single line, I would define msg closer to the `assert`. It helps when reading pytest trace